notebooks/curation_api/python/get_collections.ipynb

```diff
@@ -0,0 +1,129 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c5490d93",
+   "metadata": {},
+   "source": [
+    "# Fetch Collections index with Dataset previews"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "74560383",
+   "metadata": {},
+   "source": [
+    "The script in this notebook fetches... \n",
+    "- all public Collections (with query param `visibility=PUBLIC` or not set).\n",
+    "\n",
+    "or\n",
+    "- all access-allowed private Collections (with query param `visibility=PRIVATE`; access token required).\n",
+    "\n",
+    "In order to fetch private Collections with this script, you must have a Curation API key (obtained from upper-righthand dropdown in the Data Portal UI after logging in). You may fetch public Collections without an API key / access token."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "928aa50e",
+   "metadata": {},
+   "source": [
+    "#### <font color='#bc00b0'>Please fill in the required values:</font>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2c9a0189",
+   "metadata": {},
+   "source": [
+    "<font color='#bc00b0'>(Required) Provide the path to your api key file</font>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a4b6087b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "api_key_file_path = \"path/to/api-key-file\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0009d2d0",
+   "metadata": {},
+   "source": [
+    "### Import dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "aadd3c12",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from src.utils.config import set_api_access_config\n",
+    "from src.utils.logger import set_log_level  # can set_log_level(\"ERROR\") for less logging; default is \"INFO\"\n",
+    "from src.collection import get_collections"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9501afeb",
+   "metadata": {},
+   "source": [
+    "### Set url and access token env vars"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a6aab8e0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "set_api_access_config(api_key_file_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6085d322",
+   "metadata": {},
+   "source": [
+    "### Fetch Collection"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cb3dd36b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "collections = get_collections()"
```

Comment:
not necessary but might be nice to add a line or comment with 
`private_collections= get_collections(visibility="PRIVATE")`